### DISCUSSION TITLE
Welcome to internet_archive_downloader Discussions!

### DISCUSSION THREAD

**Discussion by elementdavv** [elementdavv/internet_archive_downloader#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by Mazefaka** [comment]
How do I download the book in its original, maximum quality?
Even when I choose 4 stars (isn't that the maximum quality level?), the final size is several times smaller than the original size stated in the description.

**Reply by AnxiousPleaseHelp** [reply]
> How do I download the book in its original, maximum quality? Even when I choose 4 stars (isn't that the maximum quality level?), the final size is several times smaller than the original size stated in the description.

Just select 4 stars. This is the highest quality available. I don't think it's possible to get anything higher.
